montrek-software/montrek
# 236 add montrek and client logo

## Background

- The montrek logo is added in the left hand corner and a client logo can be added in the right hand corner

## How to test

- The montrek logo should apper per se:

![image](https://github.com/chrishombach/montrek/assets/29869670/41d239c0-fd8a-4034-aa85-b2bda40e9321)

- Add a image to `baseclasses/static/logos` and add
```
CLIENT_LOGO_PATH=<logoname>
```
to `.env`

![image](https://github.com/chrishombach/montrek/assets/29869670/fc7d0e75-b0d9-4dd4-b250-17cb85afb36b)
